cmd/rook/main.go

```diff
@@ -232,6 +240,14 @@ func setLogLevel() {
 	capnslog.SetGlobalLogLevel(cfg.logLevel)
 }
 
+func logStartupInfo(cmdFlags *pflag.FlagSet) {
+	// log the version number, arguments, and all final flag values (environment variable overrides
+	// have already been taken into account)
+	flagValues := flags.GetFlagsAndValues(cmdFlags, "secret")
```

Comment:
the current solution works for all the flags we currently have defined.  Do you feel strongly about enhancing the solution further right now to deal with flags we may define in the future?